includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -531,12 +555,45 @@ private function get_courses( $args ) {
 		if ( isset( $args['search'] ) ) {
 			$course_args['s'] = $args['search'];
 		}
-
+		add_filter( 'posts_clauses', [ $this, 'add_lesson_completion_count_to_course_query' ] );
 		// Using WP_Query as get_posts() doesn't support 'found_posts'
-		$courses_query     = new WP_Query( apply_filters( 'sensei_analysis_overview_filter_courses', $course_args ) );
+		$courses_query = new WP_Query( apply_filters( 'sensei_analysis_overview_filter_courses', $course_args ) );
+		remove_filter( 'posts_clauses', [ $this, 'add_lesson_completion_count_to_course_query' ] );
```

Comment:
Just adding a mental note here that at some point this might have to be combined with the [days to completion filter](https://github.com/Automattic/sensei/pull/4906/files#diff-6ce1dc075c4832d5bf050e621bcfecbab91d97504e8bc565f4f476b1ea2f654dR544-R547).

Reply:
Hi, I'm not sure how to do that :(